templates.go

```diff
@@ -453,7 +453,8 @@ write_files:
   encoding: gzip+base64
   content: {{.TLSAssets.EtcdServerKey}}
 
-{{range .Files}}- path: {{.Metadata.Path}}
+{{range .Files}}
+- path: {{.Metadata.Path}}
```

Comment:
This newline is needed or the 2nd file has incorrect indentation in the cloudconfig. This means that the file contents is wrong.